third_party/move/move-model/src/model.rs

```diff
@@ -1675,6 +1675,48 @@ impl GlobalEnv {
         data.def = Some(def);
     }
 
+    /// Adds a new function definition.
+    pub fn add_function_def(
+        &mut self,
+        module_id: ModuleId,
+        name: Symbol,
+        loc: Loc,
+        visibility: Visibility,
+        type_params: Vec<TypeParameter>,
+        params: Vec<Parameter>,
+        result_type: Type,
+        def: Exp,
+    ) {
+        let called_funs = def.called_funs();
```

Comment:
Would we want to extend `called_funs()` to include closures that can be called as well (it is currently limited to move functions called)? Or maybe we expect this to be implemented when full support for lambda is added?

Reply:
Actually a good catch, I added this now to `called_funs()` so it is not forgotten.